whois-api/src/test/java/net/ripe/db/whois/api/fulltextsearch/ElasticFullTextSearchTestIntegration.java

```diff
@@ -1745,7 +1739,7 @@ public void search_organisation_with_exact_match_multiple_object_type() {
         rebuildIndex();
         final QueryResponse queryResponse = query("facet=true&format=xml&hl=true&q=(remarks:(secondDomain%20nl))" +
                 "+AND+" +
-                "(object-type:organisation+OR+object-type:person)&start=0&wt=json");
+                "(object-type:organisation+OR+object-type:person)&start=0&wt=json&rows=3");
         assertThat(queryResponse.getResults().getNumFound(), is(1L));
```

Comment:
why you need to add rows as 3?

Reply:
I was testing the limit, then I see that there was already a test for that and I forgot to delete the rows from the parameter.
So I will delete that rows